packages/synapse-react-client/src/components/index.ts

```diff
@@ -0,0 +1,58 @@
+export * from './AccessTokenPage'
```

Comment:
If you want to export a new component, make a new folder and export the contents of that folder here